Tests/LibGfx/TestICCProfile.cpp

```diff
@@ -274,6 +274,32 @@ TEST_CASE(sRGB_to_lab)
     EXPECT_APPROXIMATE_LAB(lab_from_sRGB(255, 255, 255), expected[7]);
 }
 
+static void test_roundtrip(Gfx::ICC::Profile const& profile)
+{
+    // Ideally this would be 1, but that makes tests take a few minutes on fast machine.
```

Comment:
We could use `GEN` to test a different subset on every run. Which means that we should have complete coverage over time.
I'm just thinking out-loud.